packages/gatsby-source-wordpress/src/gatsby-browser.ts

```diff
@@ -62,20 +74,13 @@ function hydrateImages(): void {
             hydrationData.innerHTML
           )
 
-          // @ts-ignore - createRoot is on ReactDOM
           if (ReactDOM.createRoot) {
-            // @ts-ignore - createRoot is on ReactDOM
-            const root = ReactDOM.createRoot(image.parentNode.parentNode)
-            // @ts-ignore - not same as below, not sure why it's complaining
-            root.render(React.createElement(mod.default, imageProps), {
```

Comment:
not sure where mod.default came from - it's undefined

Reply:
Also root.render doesn't take a second arg anymore. I think that was from before 18 was finalized